.github/workflows/pr_push.yml

```diff
@@ -15,87 +15,5 @@ permissions:
   contents: read
 
 jobs:
-  CodeStyle:
```

Comment:
Do you intend this?

Reply:
haha, rather so😄

I'm testing the GPU workflow only